rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/ReceiveMessageDirectives.scala

```diff
@@ -127,7 +128,11 @@ trait ReceiveMessageDirectives {
                 <RequestId>{EmptyRequestId}</RequestId>
               </ResponseMetadata>
             </ReceiveMessageResponse>
-          }
+            }
+          case Left(sqsError) =>
```

Comment:
isn't there a mechanism which converts all sqs error exceptions (failued futures) into xml?

Reply:
Yes, there is, but I didn't use it here since the value in the Left is actually elasticMQException.
I renamed the variable to reflect that